### DISCUSSION TITLE
will there be 1.14.7 Release this year? is there any change in plan to release 1.14.7

### DISCUSSION THREAD

**Discussion by evil-user** [dogecoin/dogecoin#3363] [General]
Just curious if there will be 1.14.7 release this year for dogecoin wallet or is there any change in plan as we have only a month and a half left for next month @patricklodder .

**Comment by patricklodder** [comment]
I'd honestly give it a 50/50 chance it'll be pulled into the next year, not the least because historically not much happens in the last 2 weeks of December.

What specific functions that are currently staged in 1.14.7-dev or in pull requests for 1.14 are you waiting for?
